pyspecProcScripts/integral_w_error.py

```diff
@@ -43,23 +44,25 @@ def integral_w_errors(s,sig_path,error_path, indirect='vd', direct='t2',fl=None,
              +str(extra_dims))
     collected_variance = ndshape(
          [ndshape(s)[indirect],len(error_path)],[indirect,'pathways']).alloc()
+    # this is not averaging over all the pathways!!!!
```

Comment:
this will need to be fixed and demonstrated on the example -- it's a bit beyond this pull request, and why I opened #44